### DISCUSSION TITLE
PKNCA Parameter differences in 0.10.2.9

### DISCUSSION THREAD

**Discussion by john-harrold** [humanpred/pknca#291] [Q&A] [answered]
I was testing ruminate out with the latest version and my internal messages flagged that vd.obs and vd.pred were no in the list of valid parameters. I get those from the column names in `PKNCA::get.interval.cols()`, I'm just curious about why those parameters were dropped. They would seem relatively useful. 

These are the differences between the current version on github and the release on CRAN:

```
> setdiff(pknca_params_10_2, pknca_params_10_2_9)
[1] "vd.obs"  "vd.pred"
> setdiff(pknca_params_10_2_9, pknca_params_10_2)
[1] "cav.int.last"     "cav.int.all"      "count_conc"       "totdose"          "cav.int.inf.obs" 
[6] "cav.int.inf.pred"
```

**Comment by billdenney** [comment, marked as answer]
Vd was mathematically a duplicate of Vz, so it was removed.

`count_conc`  and `totdose` were mostly added to allow quality checks.

**Reply by john-harrold** [reply]
That makes sense thanks.
